tests/test_deriv_api_calls.py

```diff
@@ -10,10 +10,10 @@ async def send(self, args):
 async def test_deriv_api_calls(mocker):
     api = DerivedDerivAPICalls()
     assert isinstance(api, DerivAPICalls)
-    assert (await api.account_closure({'account_closure': 1, 'reason': 'want'})) == {'account_closure': 1,
-                                                                                    'reason': 'want'}, 'account_closure can get right result'
-    with pytest.raises(ValueError, match='Required parameters missing: reason'):
-        await api.account_closure({})
+    assert (await api.exchange_rates({'exchange_rates': 1, 'base_currency': 'USD'})) == {'exchange_rates': 1,
```

Comment:
is this update automatically by some script?

Reply:
This is a test fixed manually by me.